src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteClient.java

```diff
@@ -38,6 +41,7 @@
     // Map containing all the challenges per project
     private final Map<String, Project> projects;
     private final Map<Long, Map<String, Challenge>> challenges;
+    private static Optional<String> outputPath = Optional.empty();
```

Comment:
Should this be static? What if we wanted different MapRouletteClients to post output in different places?

Reply:
Initially made this a s a static variable so that I test the method I created easily. I modified it now and its not static anymore.